app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,74 @@
+class QuestionsController < ApplicationController
+  before_action :set_question, only: [:show, :edit, :update, :destroy]
+
+  # GET /questions
+  # GET /questions.json
+  def index
+    @questions = Question.all
+  end
+
+  # GET /questions/1
+  # GET /questions/1.json
+  def show
+  end
+
+  # GET /questions/new
+  def new
+    @question = Question.new
+  end
+
+  # GET /questions/1/edit
+  def edit
+  end
+
+  # POST /questions
+  # POST /questions.json
+  def create
+    @question = Question.new(question_params)
+
+    respond_to do |format|
+      if @question.save
+        format.html { redirect_to @question, notice: 'Question was successfully created.' }
```

Comment:
Line is too long. [91/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.